test/test.spec.js

```diff
@@ -33,6 +33,27 @@ describe("replace object values based on their keys", function() {
         expect(str).to.equal("My username is tjcafferkey on GitHub");
     });
 
+    it("replace object values based on one nested key", function() {
+      var str = stringInject("My username is {profile.username}", { profile: { username: "tjcafferkey" } });
+      console.log(str)
```

Comment:
Can you remove this console.log please

Reply:
Yes. Removed the unnecessary log in the commit [c0698b0](https://github.com/tjcafferkey/stringinject/pull/10/commits/c0698b09993ec6e4f757417a8ce5e61124e29289)